lab-06-lindsay/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Don't forget to add all dependencies (ie - chai) so that when someone pulls your repo down, all they have to do is an npm install and they'll have all they need to run your program and test it!